solidity/contracts/system/KeepFactorySelection.sol

```diff
@@ -89,6 +89,26 @@ library KeepFactorySelection {
         return factory;
     }
 
+    /// @notice Sets the minimum bondable value required from the operator to
+    /// join the sortition pool for tBTC.
+    /// @param _minimumBondableValue The minimum unbonded value allowing
+    /// an operator to join and stay in the sortition pool for the application.
+    function setMinimumBondableValue(
+        Storage storage _self,
+        uint256 _minimumBondableValue
```

Comment:
Should we compute this rather than pass it in? i.e., rather than making it a setter, make it literally a `refresh` that takes the lowest lot size and multiplies it by the current price, or some factor thereof?